README.md

```diff
@@ -27,3 +27,11 @@ about the current recursive call.
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. You do not need to prove that the invariant is correct. Add
 your answer to this markdown file.
+
+Invariant: For each recursive call to fibHelper with a given n, all elements in the array a at indicies less than n will be defined if they are still needed for the computation of arr[n].
```

Comment:
So the elements just need to be defined? Or do they need to have specific values?